RealmSwift/List.swift

```diff
@@ -314,14 +314,16 @@ public final class List<T: Object>: ListBase {
 
      - parameter index: The index at which to remove the object.
      */
-    public func remove(objectAtIndex index: Int) {
+    public func remove(at index: Int) {
```

Comment:
It's sort of gratuitous overhead for something that's at best vaguely convenient, but matching the standard library is probably the way to go.